src/precice/config/ParticipantConfiguration.hpp

```diff
@@ -119,7 +112,12 @@ class ParticipantConfiguration : public xml::XMLTag::Listener
 
   mapping::PtrMapping getMapping ( const std::string& mappingName );
 
-  void finishParticipantConfiguration ( const impl::PtrParticipant& participant );
+  /// does the config already provide a solution for a parallel participant (master or server)
+  bool _isParallelSolutionDefined = false;
```

Comment:
I don't understand what this is supposed to mean.

Reply:
I added more explanation in the comment.